README.md

```diff
@@ -129,7 +129,7 @@ See [this DSP blog post](https://broadworkbench.atlassian.net/wiki/x/AoGlrg) for
 ### Running the end-to-end tests
 
 The end-to-end test is specified in `.github/workflows/run-e2e-tests.yaml`. It calls [the test script defined 
-in the dsp-reusable-workflows repo](https://github.com/broadinstitute/dsp-reusable-workflows/blob/main/e2e-test/tsps_e2e_test.py).
+in the dsp-reusable-workflows repo](https://github.com/broadinstitute/dsp-reusable-workflows/blob/main/e2e-test/teaspoons_e2e_test.py).
```

Comment:
current path is `https://github.com/broadinstitute/dsp-reusable-workflows/blob/main/e2e-test/tsps_gcp_e2e_test.py`; we should change it in that repo and here to `https://github.com/broadinstitute/dsp-reusable-workflows/blob/main/e2e-test/teaspoons_gcp_e2e_test.py`